### PR TITLE
[ADMINAPI-772] Multitenancy E2E Testing

### PR DESCRIPTION
**Pull request by simpat-adam**

### CODE REVIEW

File: Application/EdFi.Ods.AdminApi/E2E Tests/Multitenant/Admin API E2E 2.0 - Actions.postman_collection.json [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+										"exec": [
+											""
+										],
+										"type": "text/javascript"
+									}
+								}
+							],
+							"request": {
+								"method": "GET",
+								"header": [
```
Comment (CSR2017) [inline Application/EdFi.Ods.AdminApi/E2E Tests/Multitenant/Admin API E2E 2.0 - Actions.postman_collection.json:79]: @simpat-adam 
Please set the tenant specific request header on collection level. So, that we can avoid updating individual endpoints.

File: Application/EdFi.Ods.AdminApi/E2E Tests/Admin API.postman_environment.json [hunk 2]
```diff
@@ -14,18 +14,30 @@
...
+			"value": "Tenant1",
+			"type": "default",
+			"enabled": true
+		},
+		{
+			"key": "Tenant2",
+			"value": "Tenant2",
+			"type": "default",
+			"enabled": true
+		}
```
Comment (CSR2017) [inline Application/EdFi.Ods.AdminApi/E2E Tests/Admin API.postman_environment.json:34]: @simpat-adam 
Do we need tenant details here ?

Comment (simpat-adam) [reply Application/EdFi.Ods.AdminApi/E2E Tests/Admin API.postman_environment.json:34]: Removed.

File: Application/EdFi.Ods.AdminApi/E2E Tests/Multitenant/Admin API E2E 2.0 - Actions.postman_collection.json (hunk 1 above)
Comment (simpat-adam) [reply Application/EdFi.Ods.AdminApi/E2E Tests/Multitenant/Admin API E2E 2.0 - Actions.postman_collection.json]: Adjusted.

File: .github/workflows/api-e2e-multitenant.yml [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+        run: |
+          docker compose \
+            -f '../../Docker/Compose/pgsql/MultiTenant/compose-build-dev-multi-tenant.yml' \
+            --env-file './E2E Tests/gh-action-setup/.automation.env' \
+            up -d
+
+      - name: Verify containers are running
+        run: |
+          chmod +x './E2E Tests/gh-action-setup/inspect.sh'
+          './E2E Tests/gh-action-setup/inspect.sh' adminapi-packaged
```
Comment (CSR2017) [inline .github/workflows/api-e2e-multitenant.yml:57]: @simpat-adam 
We can change the container name to adminapi on docker file "\Docker\Compose\pgsql\MultiTenant\compose-build-dev-multi-tenant.yml".

**Review by CSR2017 — APPROVED** [review]

**Pull request merged**